pkg/util/gpu/devs.go

```diff
@@ -0,0 +1,34 @@
+package gpu
```

Comment:
Need a copyright header

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.

package gpu
```